src/components/Header/AccountMenu/index.tsx

```diff
@@ -0,0 +1,20 @@
+import AvatarPlaceholder from "@/assets/Images/Avatar.jpg";
+import { Menu } from "lucide-react";
+import Image from "next/image";
+import styles from "./account-menu.module.css";
+
+const AccountMenu = () => {
```

Comment:
You could use destructuring and create a default value when this data doesn't have filled

Example:

```
export default (props) => {
    const { username = "Kadoodle" } = props
    return (
        <h1>Hello {username}!</h1>;
    )
}
```

This will ensure that this component could be extensible

Reply:
@edmolima 
just a question, if the solution was like this:

```
export default ({username = "Kadoodle" } : IAccountMenu) => {
    return (
        <h1>Hello {username}!</h1>;
    )
}
```

Could work?

Reply:
Hey @guim0, yes this is absolutely work

I told to you use like receive "props" because I use this notation:

```
export const Example: FC<IExampleProps> = (props) => {
}
```

But anyway, you can use like this

```
export default ({username = "Kadoodle" } : IAccountMenu) => {
    return (
        <h1>Hello {username}!</h1>;
    )
}
```

I mean, do not use just export default anonymous function, is prefere to use export default NameComponent
this should be work better